frontend/src/components/DocumentStepper/DocumentStepperStep.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import classNames from "classnames";
+import { formatDistanceToNow, parseISO } from "date-fns";
+import {
+  FontAwesomeIcon,
+  FontAwesomeIconProps
+} from "@fortawesome/react-fontawesome";
+import { faCheckCircle, faDotCircle } from "@fortawesome/free-solid-svg-icons";
+import classes from "./DocumentStepper.module.scss";
+
+interface IconProps extends Omit<FontAwesomeIconProps, "icon"> {
+  complete?: boolean;
+}
+
+const StatusIcon: React.FC<IconProps> = ({ complete, ...rest }) => (
+  <FontAwesomeIcon {...rest} icon={complete ? faCheckCircle : faDotCircle} />
+);
```

Comment:
Instead of having to omit the type and then passing in complete, why not just extend it, add complete, and then simply toggle between two <FontAwesomeIcon/>?

Reply:
disregard.